models/genmodels.go

```diff
@@ -308,7 +308,7 @@ var queryParamStr = `
 func generateOutputs(packagePath string, s spec.Swagger) error {
 	g := swagger.Generator{PackagePath: packagePath}
 
-	g.Printf("package models\n\n")
+	g.Printf("package models\nimport \"fmt\"\n")
```

Comment:
This should have doubled `\n` to be properly formatted.

Reply:
👍